.github/workflows/tests_lgpumpi_python.yml

```
@@ -186,4 +189,4 @@ jobs:
         run: |
           rm -rf ${{ steps.setup_venv.outputs.venv_name }}
           rm -rf * .git .gitignore .github
-          pip cache purge
+          pip cache purge
```

To fix the rendering issue,
```suggestion
          pip cache purge

```